spec/rails_helper.rb

```diff
@@ -46,7 +46,7 @@ def sign_in(resource, _deprecated = nil, scope: nil)
   config.include ActiveSupport::Testing::TimeHelpers
 
   config.before :each, type: :feature do
-    https = ENV['LOCAL_HTTPS'] == 'true'
+    https = Rails.env.production? || ENV['LOCAL_HTTPS'] == 'true'
```

Comment:
I'm pretty sure this is always the "test" environment though, in spec/rails_helper.rb 🤔